### DISCUSSION TITLE
Get Attribute properties

### DISCUSSION THREAD

**Discussion by PablitoFott14** [mozman/ezdxf#866] [Issues]
Hi,

I was trying to get some text stored inside an Attrib tag. This is the part of the code im trying fot this,  but is not working:
```
`for entity in block:
     if entity.dxftype() == 'ATTRIB':
            tag = entity.dxf.tag
            tag_values.append(tag)`
```

Thanks Sir

**Comment by mozman** [comment]
https://ezdxf.mozman.at/docs/howto/content.html#howto-get-attribs
